apps/wallet-mobile/src/features/Receive/useCases/DescribeSelectedAddressScreen.tsx

```diff
@@ -40,12 +44,30 @@ export const DescribeSelectedAddressScreen = () => {
     copy(selectedAddress)
   }
 
+  const handleOnModalConfirm = React.useCallback(
+    (method: AddressMode) => {
+      if (method === 'multiple') {
+        navigate.replaceReceiveMultiple()
```

Comment:
to keep consistency with hook's name
```suggestion
        navigateTo.replaceReceiveMultiple()
```